### DISCUSSION TITLE
1:1 printout of the geometry of the bow modell

### DISCUSSION THREAD

**Discussion by PetersFozzy** [bow-simulation/virtualbow#258] [Ideas]
I would like to have a possibility to create a 1:1 printout in different formats of the geometrical data of the created bow.

**Comment by stfnp** [comment]
Yes, that would be cool. The easiest way is probably to export the shapes to a pdf file.

This is somewhat related to https://github.com/bow-simulation/virtualbow/issues/256 and https://github.com/bow-simulation/virtualbow/issues/91, but not exactly the same, so I will create a new issue.

**Reply by PetersFozzy** [reply]
Is there a way to export the shapes to a pdf file? I'm trying my hand at various models here and naturally want to utilize the results in some way. PDF would be ok, but of course an Excel matrix or .dxf or .svg would be easier
